src/sentry/static/sentry/app/actionCreators/prompts.tsx

```diff
@@ -1,10 +1,16 @@
 import {Client} from 'app/api';
 
 type PromptsUpdateParams = {
+  /**
+   * The numeric organization ID
```

Comment:
This comment is confusing if you don't have context, it says it's a numeric id but it's typed as a string

Reply:
I'll add `as a string` the end. We should try and audit this stuff and see how hard it would be to cleanup some of the naming and typing :/